lib/tbd/psi.rb

```diff
@@ -2297,9 +2297,8 @@ def process(model = nil, argh = {})
 
             next unless match
 
-            parapets = edge[:psi].select {|ty| ty.to_s.include?("parapet")}
-            roofs    = edge[:psi].select {|ty| ty.to_s.include?("roof")}
-            type     = nil
+            parapets = edge[:psi].keys.select {|ty| ty.to_s.include?("parapet")}
+            roofs    = edge[:psi].keys.select {|ty| ty.to_s.include?("roof")}
```

Comment:
This part of `psi.rb` catches instances of (in sequence) **story-**, **spacetype-**, **space-**, or **surface-specific** :roof/:parapet overrides. No RSpec was testing this specifically ... and it wasn't working. This is 1 or 2 (simple) errors.